website/docs/quickstart.mdx

```diff
@@ -6,75 +6,68 @@ import Tabs from '@theme/Tabs';
 
 import TabItem from '@theme/TabItem';
 
-This guide helps you set up Unleash Enterprise in your own environment using [Docker](#set-up-unleash-with-docker). Alternatively, you can sign up for a [cloud-hosted trial instance](https://www.getunleash.io/pricing).
+The easiest way to get started with Unleash is through a [cloud-hosted free trial](https://www.getunleash.io/plans/enterprise-payg). This gives you a ready-to-use instance, so you can explore all Unleash features without any local setup.
 
-If you are looking to set up the open-source version of Unleash, please refer to the instructions in the [Unleash GitHub repository](https://github.com/Unleash/unleash?tab=readme-ov-file#get-started-with-unleash).
+In this guide, you'll:
 
-## Set up Unleash with Docker
+1. **Sign up** for a free, hosted Unleash Enterprise instance.
 
-To start Unleash locally, clone the Unleash repository and start the server with [Docker Compose](https://docs.docker.com/compose/):
+2. **Create your first feature flag** using the Unleash Admin UI.
 
-```shell
-git clone git@github.com:Unleash/unleash.git
+3. **Connect Unleash to your application** by integrating an SDK into your codebase.
 
-cd unleash
-docker compose -f docker-compose-enterprise.yml up -d
-```
-
-This pulls the `unleashorg/unleash-enterprise` Docker image and uses a Docker Compose file to configure the Unleash server and its database.
-
-> This step uses `docker compose` (V2 syntax). If you have the older `docker-compose` (V1), use that command syntax instead.
-
-## Log in to the Unleash Admin UI
-
-In your browser, go to [http://localhost:4242](http://localhost:4242) and log in using the following credentials:
-- **username**: `admin`
-- **password**: `unleash4all`
-
-![Unleash Admin UI log in screen](/img/quickstart-login.png)
-
-## Install your trial license
-
-Request a license by signing up for a self-hosted trial [here](https://www.getunleash.io/pricing). Once you've signed up, you'll receive an email from Unleash containing your trial license key.
+![The Unleash Admin UI onboarding experience](/img/onboarding-experience.png)
 
-In the Admin UI, go to **Admin settings > Instance config > License**, copy the license key you received by email and click **Update license key**.
+If you prefer to manage your own infrastructure, see the [self-hosted setup guide](./using-unleash/deploy/getting-started). To use the open-source version, see the instructions in the [Unleash GitHub repository](https://github.com/Unleash/unleash?tab=readme-ov-file#get-started-with-unleash).
 
-The top banner now displays the number of days you have left on your free trial.
+## Sign up for an Unleash Enterprise Cloud trial
```

Comment:
Should we label this 1. , to correspond to the three steps people will be taking?   Will let you decide but I think it might make it clearer.